pkg/resources/db/resource_db_acceptance_test.go

```diff
@@ -1,4 +1,4 @@
-package resources_test
```

Comment:
Package names should not contain dashes or underscores

Reply:
Using _test for the package of test is a valid strategy: 
[https://pkg.go.dev/cmd/go@master#hdr-Test_packages](https://stackoverflow.com/questions/19998250/proper-package-naming-for-testing-with-the-go-language/31443271#31443271)
[https://pkg.go.dev/cmd/go@master#hdr-Test_packages](https://pkg.go.dev/cmd/go@master#hdr-Test_packages)